tests/screenshot/apps/components/Image.js

```diff
@@ -0,0 +1,9 @@
+import Image from '../../../../Image';
+import React from 'react';
+
+const ImageTests = [
+	<Image />,
+	<Image src="http://via.placeholder.com/200x200" />,
+	<Image sizing="fit" src="http://via.placeholder.com/200x200" />
+];
+export default ImageTests;
```

Comment:
We need one empty line before defining `export default` and the last line in a file.